cf-monitord/mon_network.c

```diff
@@ -202,42 +218,288 @@ void MonNetworkGatherData(double *cf_this)
 {
     ResetNetworkData();
 
-    char comm[PATH_MAX + 3] = {0};
-    strncpy(comm, VNETSTAT[VSYSTEMHARDCLASS], (sizeof(comm) - 1));
+    Item *in[ATTR] = {0};
+    Item *out[ATTR] = {0};
 
-    if (!FileCanOpen(comm, "r"))
+#ifdef __linux__
+    if ((access("/proc/net/tcp", R_OK) != 0) || !GetNetworkDataFromProcNet(cf_this, in, out))
```

Comment:
Deserves a comment. For example: `On linux, prefer procfs, fall back to netstat if proc parsing fails`